docs/NextJs.md

```diff
@@ -14,6 +14,45 @@ Next.js 13 proposes 2 ways to build a React project:
 
 React-admin supports both ways. 
 
+## Prevent SSR
+React-admin is designed as a Single-Page Application, rendered on the client-side. It comes with its own routing sytem, which conflicts with the Next.js routing system. So we must prevent Next.js from rendering the react-admin component on the server-side. 
+
+To do that, we will have to create a component to bootstrap the `<Admin>` application, and import it in Next.js by using the [__lazy loading__ system provided by Next.js](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading) and specify the [`ssr` option to false](https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#with-no-ssr). 
+
+Using dynamic import allows disabling Server-Side Rendering for the `<Admin>` component.
```

Comment:
You're saying the same in this sentence as in the previous sentence. Or am I missing something?